dataflow/flex-templates/streaming_beam/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM gcr.io/dataflow-templates-base/python3-template-launcher-base
+
+RUN mkdir -p /cloud/dataflow
+
+COPY streaming_beam.py /cloud/dataflow
```

Comment:
Nit: if we're declaring the requirements env variable first, I would follow the same order when copying.

Reply:
Done.